flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -1185,7 +1185,7 @@ abstract class TableTestUtil(
       name)
     val operation = new RichTableSourceQueryOperation(identifier, tableSource, statistic)
     val table = testingTableEnv.createTable(operation)
-    testingTableEnv.registerTable(name, table)
+    testingTableEnv.createTemporaryView(name, table)
```

Comment:
There is a TODO comment above mentioning `registerTable`. Shouldn't it be updated as well?

Reply:
Good catch. I will update the comment.